app/services/report_importer.rb

```diff
@@ -44,6 +59,15 @@ def import
     end
   end
 
+  def scan
+    logger.info "Scanning report with: #{self.class.report_scanner.join(', ')}"
+    self.class.report_scanner.each do |scanner|
```

Comment:
do we need to go over each scanner? I guess the `scanner.scan` should return true/false (or string/nil) and we should use `any?` instead of `each`.

Reply:
What would the return of `scanner.scan` indicate? 

Defining more narrowly what the scan method of registered scanners is able/allowed to do might help. ATM it has free reign over what it does to the report and it's associations.